SECURITY.md

```diff
@@ -7,3 +7,6 @@ To report a security issue, email [electron@github.com](mailto:electron@github.c
 The Electron team will send a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
```

Comment:
It would be really useful to have a time limit here, like "The team will respond in 48 hours to confirm that someone has seen your message. If you do not get a response, do X."

Reply:
That would be useful, but no such policy exists right now and it would need to go through the maintainers group, so no need to block this PR on it. @diracdeltas you may want to open a separate issue for that suggestion.